<ui version="4.0" >
<comment> Begin CVS Header </comment>
<comment>   $Source: /Volumes/Home/Users/shoops/cvs/copasi_dev/copasi/UI/CQEventWidget1.ui,v $ </comment>
<comment>   $Revision: 1.5 $ </comment>
<comment>   $Name:  $ </comment>
<comment>   $Author: shoops $ </comment>
<comment>   $Date: 2009/04/27 00:03:17 $ </comment>
<comment> End CVS Header </comment>

<comment> Copyright (C) 2008 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., EML Research, gGmbH, University of Heidelberg, </comment>
<comment> and The University of Manchester. </comment>
<comment> All rights reserved. </comment>

 <class>CQEventWidget1</class>
 <widget class="CopasiWidget" name="CQEventWidget1" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>443</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="caption" stdset="0" >
   <string>Event Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="mpLabelEventName" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Event Name</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3" >
      <widget class="QLineEdit" name="mpLineEditName" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="mpLabelTrigger" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Condition Trigger</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3" >
      <widget class="CQExpressionMmlWidget" native="1" name="mpExpressionTrigger" >
       <property name="minimumSize" >
        <size>
         <width>380</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32767</width>
         <height>32767</height>
        </size>
       </property>
       <property name="mouseTracking" >
        <bool>false</bool>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="acceptDrops" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="mpLabelDelay_2" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Delay</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QRadioButton" name="mpBtnDelayNone" >
       <property name="text" >
        <string>None</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2" >
      <widget class="QRadioButton" name="mpBtnDelayCalculation" >
       <property name="text" >
        <string>Calculation</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3" >
      <widget class="QRadioButton" name="mpBtnDelayAssignment" >
       <property name="text" >
        <string>Assignment</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="mpLabelDelay" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Delay Expression</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3" >
      <widget class="CQExpressionMmlWidget" native="1" name="mpExpressionDelay" >
       <property name="minimumSize" >
        <size>
         <width>380</width>
         <height>100</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::StrongFocus</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="mpLine2_2" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" >
     <item row="1" column="0" >
      <layout class="QHBoxLayout" >
       <item>
        <widget class="Q3ListBox" name="mpLBTarget" />
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QToolButton" name="mpBtnSelectObject" >
           <property name="maximumSize" >
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>edit target</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset>
             <normaloff>image0</normaloff>image0</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="mpBtnAddTarget" >
           <property name="maximumSize" >
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>add target</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset>
             <normaloff>image1</normaloff>image1</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="mpBtnDeleteTarget" >
           <property name="maximumSize" >
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>delete target</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset>
             <normaloff>image2</normaloff>image2</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacerTargetIcons" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="1" >
      <widget class="CQExpressionMmlWidget" native="1" name="mpExpressionEA" >
       <property name="minimumSize" >
        <size>
         <width>300</width>
         <height>60</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::StrongFocus</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="mpLabelTarget" >
         <property name="text" >
          <string>Target</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacerTarget" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="1" >
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="mpLabelEA" >
         <property name="text" >
          <string>Expression</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacerExpression" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="mpLine2" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="mpBtnCommit" >
       <property name="text" >
        <string>Commit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnRevert" >
       <property name="text" >
        <string>Revert</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnNew" >
       <property name="text" >
        <string>New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnDelete" >
       <property name="text" >
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>Q3ListBox</class>
   <extends>Q3Frame</extends>
   <header>q3listbox.h</header>
  </customwidget>
  <customwidget>
   <class>CQExpressionMmlWidget</class>
   <extends>QWidget</extends>
   <header>CQExpressionMmlWidget.h</header>
  </customwidget>
  <customwidget>
   <class>CopasiWidget</class>
   <extends>QWidget</extends>
   <header>copasiWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <images>
   <image name="image0">
     <data format="PNG" length="650">89504e470d0a1a0a0000000d49484452000000100000001008060000001ff3ff610000025149444154388d8d93db53525114877f1b3717e32e8e9897220151b0cb288c3aa6654d33dda69e7de9efebc5e9b197a65131727286d120062727ec88e08140e0409ca370560f8d3427a3da8f7badef9bbdd6da8bc56231fccf298897499155d86c7ad89d59767ecfff067dd91f20e1a0817cae89625100d33178bd663c7ef62ba7ab60fdad99d2a91c0c068ec9290796ee98307cf598018a26ef8f82d5977a3a3aaac2ebb3e0c973990195aeafd40892bb0efaf0fe0492d4c474d889c5658975032f0876e326da8c0a301a2d585ceec774b8f24f5823d88a89b05a55cc2f38e00f5c84bd8d0c29cd1a949604a6d7a3e09a651dc1abd50aa94488cc8ec21fa869e0a1bd752aa5134889395465052744601c18f64469e2e10a78ee90d3e1d71c66c27e4c86ea1d78bc16273116c55e3a8bfc9905adc129b86f78e0ebb7e2b494c151721b7d8719f05a59c440df191696f41d78acbc45c2c66b6404a07d651e9ed02d1802639d789b19e9584ec05ed681ab6d050eabaaa9b79458c727c10843e80146ef472ef423ba16475154e033b9c00d662bc492d00906757b943a2920ab8b2012f06bc06a2a4e3befb690fd9c87772a88c1608071a7c30e55d56173ad44b7efba5893f7c36433a3b795060926d86423495509897401fbe903c80ac7a8ef26eebd58f93905975b6581e0354a7e14203538cd5c77c2ee7e04a76d033b6f36b1adb4d1623da8cb3d20db087cb373083f9deb94c5ceb731befd9d12c96f288a15b0b3325a8d3aead553f01e6068c482f1801b13a111b8c75d9a9eb0dfd7399f3352a3de00b575e09ca3f7920183c372d75ff9035105f9b717f5665a0000000049454e44ae426082</data>
   </image>
   <image name="image1">
     <data format="PNG" length="411">89504e470d0a1a0a0000000d49484452000000100000001008060000001ff3ff610000016249444154388d95934f4a033118c57fc9a42da58268414110bb70275e202282e84eef502fe1ce852bafa02b11a1a80b179ea1cec2b53d8016b505c5968a586df3b99899fe9ba9d40721f9c2cbcb9b375f94effb8cc29e5ae11b70e18606d2e0eff96a946b62a701da2067c35baa98c81c2320c194bf5ee2b3f9cae6720154e51f022172c6a15342d61388990fa0ff129804261618407b982408fc803db6127d5e14ac490aac674f85bebbc2eebee56675f88fa9229848519fe7991a484421644d8b5426c36dad8e7bae317d3f8b84617806f0defa21aaf607ba2ba16570cef1a535288548702c1a81ab6065f0827a65618e9c7120d0e93876d6d62995cbbc371a6c140a3cb69a68dd57d09ec6d7550c7550db80aed2bb7e1eb6acc249508a16ee2e9ee065c082023c30fe41bc3ded8995a3994b4ab2d8671bf00fe3dce43e90a129a198506012240b84c19ab446a7145e5a8dedd9e4b7d00bf601145c4965ecabf905383f6e47d8506b3e0000000049454e44ae426082</data>
   </image>
   <image name="image2">
     <data format="PNG" length="341">89504e470d0a1a0a0000000d49484452000000100000001008060000001ff3ff610000011c49444154388da592b16ac33014458f8cdb82e9da31058f9d8d28f81bf20b8640862eddfa1ddd327608f80f42c17ff02094ccdeb3063a1543da507550acca96d252fa403ca4fbcec53c5f2522fca712ff92ce4b93ce4b736a38a627bea86b83ae0d3193537ae28b3439347930f4939e02ac5bd04d6ea73f0eb09aa06bc34ba50c80ae0dac26563f4ba1c959b75000aa5fe2a62ccdfdc3b51dda77b6cf76b62faf6cbfc800583c6e294414be8133991d87dfbbe112ce8ff072e7e0c0c0994c2fc73bb4f0f3db00763bf0ebf60678edc6cf4e3b8cde821ce8dac0fe337a62bf38c801d5e00b8755a95f723082171b7b5c65a1490236070ecebee1424415222a30b95396e90d0a11f5d48670cf8c4d7c3dc8410fc45610d3831cfcb5be0040cea4e3146f5ea80000000049454e44ae426082</data>
   </image>
   <image name="image3">
     <data format="PNG" length="1002">89504e470d0a1a0a0000000d4948445200000016000000160806000000c4b46c3b000003b149444154388dad945f4c5b551cc73fe7dc4b7b4bcba0762d45c43114323599ee6192609c51d883892ce083f1718b3ebb185f8dc91e972cf39d2d2a2f1af664b6f1e0fe3863a0718969700eb0c52142da0242a1bd6d696f7bcff101585203ceb8fd9ece39f99dcff9fe7edf939f88c562ec465f5f9fe609442c161362173c3e3eae7b7a7ac8e7f36432196cdbfe4f907c3e4f2291201e8fe338cec3737357e9e8e828aded1e229d650e1f2d51754b082110124c13a4dc5ea341eb9dc284c0558a853f3ce8cb0677ef500fde7d39d2596679e326597b8e9abb85d7a770ab16ab6983ec5a05b487a70e36f0f4e10afe408d6a558310980108478dba4a1e8233990c5d474b64ed39aa3a8fe5f3317fbf81dbd70bccfeb205947632fd74f6589c1c6ea2f70d03a58ba0c1f2c9bdc1b66de3b8256a6e11cbe7e3ee1d181b590124fe2693aeee08d223c82c3a2c24b7b874bec8f26288774f7bd054504aef0dde6e99c0eb83f9fb266323cb80a27fb0958141836044605a2ee5523393371cc646fee2da37195aa35d0c0c5b4859ac03d7e91712dcaac5adab3650a3ff9d08ef7dd8404bb48869e5d958b5b87dadc4c9a1464e9f0d0326df7ebd86bd2e310cb1bf62d384d59441f2d70a070e1c60e09489929b988681bdd9cc97170bcc4c65595f71f8e0e3301337fc24a7732467831875a47f289652b0be5e4151e6d07316c1b0c0340d8ab92023e76d66a6b2840e36d2fb7a13fee632475e6edc367ea98a90fb98b7dd6310ca0328a44761582e1bab41befabcc0ec940d28bc5e93b68e064cab84e1d9beaeb48934eac1f53b01c1b000fca496aa54b61a99fcde61662a4b4b4b23d1680be9d426173e4df3602a48ea411989a4fd590f52a8fd156b05ed9d350e3defe3cfdf4b4c7ce770ea7d3fb9f520afbe1620daeee5c26735d20b9b9cfb6811a754a439e4e5c5639a4caa1e5caf586bfc0197b78702005cb9b4cae4cd3267ce8638fe964bd72b393e39d74928d242617303a756a37f284447770dcdbffc6384a05a85de1306e9a52057c7527c7131c3c42d3f475eb2303c82d4fc3276d6811db37efeb148723082d9b08f79f97c1e5729109a9a28307cc622d2d6cdf52b2b24efe548dedb00142009862cfa879ee1a71f6cec928353511472fbf4389148b0b0e0c108081412458dfe21c9f11351e67e7358595468246d1d1e5e38a6e9e851bc39d84ab502a669331dafec0d8ec7e3e8cb06e1a881d727d1ae40180a434a8c9db129a54126ad48a7358c2b4c5352c8c374bcccdab2bb37d8719cba79fab8211f9df218e0582c261e95f8bfc04f1a1e8bc5c4dfe0a190172af6a9690000000049454e44ae426082</data>
   </image>
 </images>
 <tabstops>
  <tabstop>mpLineEditName</tabstop>
  <tabstop>mpExpressionTrigger</tabstop>
  <tabstop>mpBtnDelayNone</tabstop>
  <tabstop>mpBtnDelayCalculation</tabstop>
  <tabstop>mpBtnDelayAssignment</tabstop>
  <tabstop>mpExpressionDelay</tabstop>
  <tabstop>mpLBTarget</tabstop>
  <tabstop>mpBtnSelectObject</tabstop>
  <tabstop>mpBtnAddTarget</tabstop>
  <tabstop>mpBtnDeleteTarget</tabstop>
  <tabstop>mpExpressionEA</tabstop>
  <tabstop>mpBtnCommit</tabstop>
  <tabstop>mpBtnRevert</tabstop>
  <tabstop>mpBtnNew</tabstop>
  <tabstop>mpBtnDelete</tabstop>
 </tabstops>
 <includes>
  <include location="global" >string</include>
  <include location="global" >vector</include>
  <include location="local" >copasiWidget.h</include>
  <include location="local" >MyLineEdit.h</include>
  <include location="local" >CQExpressionMmlWidget.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>mpBtnCommit</sender>
   <signal>clicked()</signal>
   <receiver>CQEventWidget1</receiver>
   <slot>slotBtnCommitClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnDelete</sender>
   <signal>clicked()</signal>
   <receiver>CQEventWidget1</receiver>
   <slot>slotBtnDeleteClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnNew</sender>
   <signal>clicked()</signal>
   <receiver>CQEventWidget1</receiver>
   <slot>slotBtnNewClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnRevert</sender>
   <signal>clicked()</signal>
   <receiver>CQEventWidget1</receiver>
   <slot>slotBtnRevertClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnAddTarget</sender>
   <signal>clicked()</signal>
   <receiver>CQEventWidget1</receiver>
   <slot>slotAddTarget()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnDeleteTarget</sender>
   <signal>clicked()</signal>
   <receiver>CQEventWidget1</receiver>
   <slot>slotDeleteTarget()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnSelectObject</sender>
   <signal>clicked()</signal>
   <receiver>CQEventWidget1</receiver>
   <slot>slotSelectObject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
